src/Libraries/Microsoft.Extensions.Diagnostics.HealthChecks.ResourceUtilization/ResourceUtilizationHealthCheck.cs

```diff
@@ -39,22 +39,22 @@ internal sealed class ResourceUtilizationHealthCheck : IHealthCheck
     public Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context, CancellationToken cancellationToken = default)
     {
         var utilization = _dataTracker.GetUtilization(_options.SamplingWindow);
-        if (utilization.CpuUsedPercentage > _options.CpuThresholds?.UnhealthyUtilizationPercentage)
+        if (utilization.CpuUsedPercentage > _options.CpuThresholds.UnhealthyUtilizationPercentage)
```

Comment:
NIT: is the field no longer nullable?

Reply:
An analyzer just flagged this as being a superfluous use of ?. Not sure why the analyzer wasn't flagging this before...